packages/cli-platform-ios/src/commands/logIOS/index.ts

```diff
@@ -6,44 +6,42 @@
  *
  */
 
-import {execFileSync, spawnSync} from 'child_process';
+import {spawnSync} from 'child_process';
 import os from 'os';
 import path from 'path';
 import {logger} from '@react-native-community/cli-tools';
-import {Device} from '../../types';
-
-function findAvailableDevice(devices: {[index: string]: Array<Device>}) {
-  for (const key of Object.keys(devices)) {
-    for (const device of devices[key]) {
-      if (device.availability === '(available)' && device.state === 'Booted') {
-        return device;
-      }
-    }
-  }
-  return null;
-}
+import listIOSDevices from '../../tools/listIOSDevices';
+import {getSimulators} from '../runIOS';
```

Comment:
Let's extract `getSimulators` to a separate file so that commands like `run-ios` and `log-ios` don't depend on each other.